triciaswift/tastevine-server
# Added create method for ingredient view

I added a create method for Ingredient View

Supported routes
`/ingredients` for a POST request will return an object of the newly created ingredient

## Steps to test
1. Pull down the `ts-create-ingredients` branch and switch to it
2. If your debugger is running, restart it
3. Open Postman
4. Use this token in the header:
```
Token c29b168a0c1e461b3ff36dd9ce15e6e1d1768597
```
5. Perform a POST to `http://localhost:8000/ingredients` with the following body:
```
{"name": "apple cider vinegar"}
```
6. Make sure the status code is a 201
